Controller/EmbedController.php

```diff
@@ -19,7 +20,12 @@ public function __construct(ClientRequestManager $manager)
 
     public function renderHierarchyAction(string $template, string $parent, string $field, int $depth = null, array $sourceFields = [], array $args = []): Response
     {
-        $hierarchy = $this->clientRequest->getHierarchy($parent, $field, $depth, $sourceFields);
+		$hierarchy = $this->clientRequest->getHierarchy($parent, $field, $depth, $sourceFields);
+		
+		//Search all active items if given emsLink of currentPae
+		if(isset($args['emsLink']) && $args['emsLink'] instanceof EMSLink) {
+		    $hierarchy->setAllActives($args['emsLink']->getOuuid());
+		}
```

Comment:
Remove the comment and rename the key 'emsLink' to 'currentPage'.
Your code $args['currentPage'] instanceof EMSLink already explains that it is an EMSlink and for the currentpage (or maybe 'activePage' is better?

Reply:
I prefer 'activeChild'

Reply:
I changed the name 3 times because I hesitated with 'currentPage'..
'activePage' is better I choose it

Reply:
ok for activeChild

Reply:
done